mpi.rst

```diff
@@ -257,6 +259,17 @@ If the host MPI is Open MPI, the definition file looks like:
 
       mpirun -np 2 -mca pml ucx -x UCX_TLS=sysv,ib apptainer exec $MY_CONTAINER ./a.out
 
+Using --sharens mode
+====================
+
+By default, with unprivileged setup, each process spawned by MPI runs in its own user namespace. This will 
```

Comment:
Change `default, with unprivileged setup` to `default in unprivileged user namespace mode`.